packages/web-twig/src/Resources/components/Accordion/AccordionHeader.twig

```diff
@@ -4,7 +4,7 @@
 {%- set _id = props.id -%}
 {%- set _for = props.for -%}
 {%- set _elementType = (props.elementType is defined) ? props.elementType : 'h3' -%}
-{%- set _isOpen = props.isOpen -%}
+{%- set _isOpen = props.isOpen | default() -%}
```

Comment:
I think that the default should be boolean with `false` as in `AccordionContent`.

Reply:
fixed